### PR TITLE
add bucket for csv uploads

### PR DESCRIPTION
**Pull request by goodviber**
### Context
We need to add a store for partnership_csv_uploads
### Changes proposed in this pull request
Add s3 bucket to terraform build
### Testing
aws-s3-bucket config should be available in cf $VCAP_SERVICES
### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

**Review by ethanmills — APPROVED** [review]
Looks very familiar, probably best to wait for another review

**Review by kicferk1 — APPROVED** [review]

File: terraform/app/modules/paas/variables.tf
Comment (kicferk1) [inline terraform/app/modules/paas/variables.tf:73]: Maybe include `bucket` in the name - all other services include the answer to 'what are you?' in their names.

**Comment by kicferk1** [comment]
> Looks very familiar, probably best to wait for another review

So machiavellian, asks to wait and merges something that causes conflicts right after 😂

**Pull request merged**
